Bottom: fe04bb2ccd7bc78282926617af35db1a0ec2ad69
Top:    6f75e0811d6f2239ab200abad239faaa4e1b7f45
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-13 20:27:21 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!






---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index b09d948..93b6073 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -310,13 +310,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
